modules/@apostrophecms/login/index.js

```diff
@@ -909,8 +910,20 @@ module.exports = {
             last: true
           }
         );
-      }
+      },
 
+      getSchema() {
+        return self.apos.user.schema
+          .filter(({ name }) => [ 'username', 'password' ].includes(name))
+          .map(field => ({
+            name: field.name,
+            label: field.label,
+            placeholder: `Enter ${field.label.replace('apostrophe:', '').toLowerCase()}`,
```

Comment:
We're passing in a phrase which usually gets parsed. I just stripped it but if there's a better way lemme know.

Reply:
Hmm. This is messy, isn't it. Our original goal here was to make this placeholder easy to override and localize server side, and now it it's server side, but it's still not possible to localize fully 😄 

So I suggest:

* Add `usernamePlaceholder` and `passwordPlaceholder` options to this module
* Default those to `apostrophe:enterUsername` and `apostrophe:enterPassword` and add the appropriate phrases to `en.json`
* Utilize those options as needed here as part of your massaging of the schema array